GVRf/Framework/settings.gradle

```diff
@@ -1,3 +1,4 @@
-//include ':vrlibrary'
-
-//project(':vrlibrary').projectDir = new File('../../oculus_vr/VRLib/vRLib')
+include ':framework'
+//include ':vrlibrary'
+
+//project(':vrlibrary').projectDir = new File('../../oculus_vr/VRLib/vRLib')
```

Comment:
It looks good to me. And, the commented out lines can be cleaned anytime, possibly when I check in some later.